clients/gitlabrepo/client.go

```diff
@@ -222,7 +221,7 @@ func (client *Client) Close() error {
 }
 
 func CreateGitlabClientWithToken(ctx context.Context, token string, repo clients.Repo) (clients.RepoClient, error) {
-	client, err := gitlab.NewClient(token, gitlab.WithBaseURL(repo.URI()))
+	client, err := gitlab.NewClient(token, gitlab.WithBaseURL(repo.Host()))
```

Comment:
Since we have access to internal members of `repoURL` struct, we can just access `repo.host` instead ?

Reply:
Do you mean by exporting the variable as `Host` instead of using a getter? I don't think we can access the private field from here.

Reply:
You should be able to access the private field of `repoURL` struct since its in the same package.